plugins/services/src/js/pages/task-details/TaskDetail.js

```diff
@@ -317,12 +318,13 @@ class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
         <div className="flex flex-item-shrink-0 control-group">
           {this.getBreadcrumbs()}
         </div>
-      <RouteHandler
-        directory={directory}
-        onOpenLogClick={this.handleOpenLogClick.bind(this)}
-        selectedLogFile={selectedLogFile}
-        service={this.getService()}
-        task={task} />
+        {this.props.children && React.cloneElement(this.props.children, {
```

Comment:
Isn't only `React.cloneElement(this.props.children, ...` enough on this case? If not perhaps you should also change the other occurrence above?